talpid-core/src/tunnel/openvpn.rs

```diff
@@ -684,12 +685,13 @@ mod event_server {
             &self,
             request: Request<EventType>,
         ) -> std::result::Result<Response<()>, tonic::Status> {
-            log::info!("OpenVPN event {:?}", request);
-
             let request = request.into_inner();
+            log::debug!("OpenVPN event {:?}", request);
```

Comment:
We should not have debug representations (`{:?}`) in an `info` level log. Debug representations should only exist in `debug` level and more detailed. I also felt we could do this logging *after* calling `into_inner` in order to not log a bunch of gRPC headers we don't care about.